api/config/cdr_config_test.json

```diff
@@ -143,6 +143,29 @@
       "microarrayVcfSingleSampleStoragePath": "microarray/vcf/single_sample",
       "accessTier": "controlled",
       "tanagraEnabled": true
+    },
+    {
+      "cdrVersionId": 9,
+      "isDefault": true,
+      "name": "Synthetic Dataset in the Controlled Tier v4",
+      "bigqueryProject": "fc-aou-cdr-synth-test-2",
+      "bigqueryDataset": "SC2023Q3R3",
+      "wgsBigqueryDataset": "1kg_wgs_2022q1",
+      "wgsFilterSetName": "echo-controls",
+      "creationTime": "2024-11-04 19:58:48Z",
+      "releaseNumber": 8,
+      "numParticipants": 234525,
```

Comment:
my understanding is that all values from here downward should match the previous CDR.  That appears to be the case ✅